### ISSUE TITLE
Due date changes not synced

### ISSUE THREAD

**Issue by stacksjb** [scottquach/Canvas-Assignments-Transfer-For-Todoist#12]
I've discovered that if an assignment due date is updated by a professor in Canvas, the tool will not sync the updated due date, since it already sees that the task has been synced.

This isn't frequent but is a problem when it occurs.

**Comment by scottquach** [comment]
I've made an update that will update the todoist item if the canvas due date is different than the todoist due date

**Comment by andreicozma1** [comment]
@scottquach 
```
if (task['due'] and task['due'].get('date') != assignment.get('due_date'))
```
Might need to be:
```
if (task['due'] and task['due'].get('date') != assignment.get('due_at')):
```

'due_date' is an invalid key therefore the result comes out as None

**Comment by andreicozma1** [comment]
Raised a PR with fix on #13
